home/terminal.nix

```diff
@@ -0,0 +1,76 @@
+{ pkgs, ... }:
+
+# Platform-independent terminal setup
+{
+  # Nix packages to install to $HOME
+  #
+  # Search for packages here: https://search.nixos.org/packages
+  home.packages = with pkgs; [
+    # Unix tools
+    ripgrep # Better `grep`
+    fd
+    sd
+    tree
+
+    # Nix dev
+    cachix
+    nil # Nix language server
+    nix-info
+    nixpkgs-fmt
+    nixci
+
+    # Publishing
+    asciinema
+
```

Comment:
Remove this, or comment it out.

Reply:
https://github.com/cameronraysmith/nix-dev-home/commit/ea2e56981e63610343970a58c9efcbffe9ff1588